content/videos/playwright-testing-modern-web-applications.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Playwright Testing for Modern Web Applications
+date: 2025-01-15
+description: Learn how to leverage Playwright for end-to-end testing in modern web applications. This session covers best practices, advanced techniques, and real-world examples to help you build robust test suites that catch bugs before they reach production.
+video: tdmgXAUfWFo
+tags: [playwright, testing, e2e, web-applications]
+image: https://res.cloudinary.com/debsobrien/image/upload/v1640019149/debbie.codes/blog/playwright-testing_g8mn2k.jpg
```

Comment:
remove tags e2e and web applications
see if you can get the correct title and description from the youtube url